src/viewProviders/schemas.ts

```diff
@@ -72,23 +72,33 @@ export class SchemasViewProvider implements vscode.TreeDataProvider<SchemasViewP
     this.treeView = vscode.window.createTreeView("confluent-schemas", { treeDataProvider: this });
 
     ccloudConnected.event((connected: boolean) => {
```

Comment:
Will we be growing a peer local event listner in the final branch for localConnection events?

Reply:
Kind of - since the local resources can be started/stopped more easily than entire CCloud environment configs, we'll have a separate event listener just for local SR availability: https://github.com/confluentinc/vscode/blob/djs/local-sr-start/src/viewProviders/schemas.ts#L86-L92